### PR TITLE
[feature-582] Update release version 1.5.1

### PR DESCRIPTION
**Pull request by xuluna**
# Description
Change the versions from 1.5.0 to 1.5.1

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/582|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

Manually tested with karavi-testing PR (https://eos2git.cec.lab.emc.com/CSM/karavi-testing/pull/194) and generated RPM version: karavi-authorization-1.5-1.x86_64.rpm

- [x] e2e test for RPM and helm installation

### CODE REVIEW

File: deploy/rpm/SPECS/karavi-authorization.spec [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 Name:           karavi-authorization
 Version:        1.5
-Release:        0
+Release:        1
```
Comment (atye) [inline deploy/rpm/SPECS/karavi-authorization.spec:17]: @sharmilarama does this need to be updated? Previous value is zero meaning we haven't updated it before.

Comment (sharmilarama) [reply deploy/rpm/SPECS/karavi-authorization.spec:17]: @xuluna Did you build the rpm using this version update? Does it build with 1.5.1 as the version - `karavi-authorization-1.5-1.x86_64.rpm`?

**Review by sharmilarama — COMMENTED** [review]
Please update manual test details in the PR description.

File: deploy/rpm/SPECS/karavi-authorization.spec (hunk 1 above)
Comment (xuluna) [reply deploy/rpm/SPECS/karavi-authorization.spec:17]: @sharmilarama Yes I tested it e2e and the build created is karavi-authorization-1.5-1.x86_64.rpm
```
$ ls deploy/rpm/x86_64
karavi-authorization-1.5-1.x86_64.rpm
```

**Comment by xuluna** [comment]
> Please update manual test details in the PR description for rpm file changes.

Updated in the description.

**Review by atye — DISMISSED** [review]

File: scripts/install_karavi_auth.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
```
Comment (alikdell) [inline scripts/install_karavi_auth.sh:2]: 2022-2023

**Review by alikdell — APPROVED** [review]

**Review by atye — DISMISSED** [review]

**Review by sharmilarama — APPROVED** [review]

**Comment by xuluna** [comment]
run e2e test

**Comment by xuluna** [comment]
run e2e test

**Comment by xuluna** [comment]
run e2e test

**Comment by xuluna** [comment]
run e2e test

**Comment by xuluna** [comment]
run e2e test

**Comment by xuluna** [comment]
I somehow forgot to sign some of the commits. I've rebased all commits into one and signed. Requesting new approvals..

**Review by alikdell — APPROVED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Comment by xuluna** [comment]
run e2e test

**Pull request merged**